searchcore/src/vespa/searchcore/proton/attribute/attributemanager.cpp

```diff
@@ -167,6 +168,7 @@ AttributeManager::transferExistingAttributes(const AttributeManager &currMgr,
                                              const Spec &newSpec,
                                              Spec::AttributeList &toBeAdded)
 {
+    std::vector<std::unique_ptr<vespalib::Gate>> gates;
```

Comment:
Consider using a single  vespalib::Gate and sending in the same vespalib::GateCallback to all tasks.

Reply:
Done